src/js/utils/dom.js

```diff
@@ -49,10 +49,10 @@ export function htmlToParentElement(html) {
 }
 
 export function sanitizeScriptNodes(element) {
-    const scripts = element.querySelectorAll('script');
-    for (let i = 0; i < scripts.length; i++) {
-        const script = scripts[i];
-        script.parentNode.removeChild(script);
+    const nodes = element.querySelectorAll('script,object,iframe');
+    for (let i = 0; i < nodes.length; i++) {
+        const node = nodes[i];
+        node.parentNode.removeChild(node);
```

Comment:
Dope implementation! Approving, but be careful of where this is utilized. If we fire it inside a 3rd party iframe (like IMA), it'll throw a cross origin iframe script blocking error.